bin/run_celery_worker_log_parser

```diff
@@ -16,8 +16,8 @@ fi
 
 exec newrelic-admin run-program celery -A treeherder worker \
     -Q \
-parse_job_logs,\
-parse_job_logs_fail,\
+parse_job_log,\
+parse_job_log_fail,\
```

Comment:
Heroku uses none of the `bin/run*` scripts, only Procfile.

However I believe the existing lines in `bin/run_celery_worker_log_parser ` were a mistake, so can just be removed entirely. The name is used only by the hjelper function, which itself isn't a celery `@task`.